Week1/homework/app.js

```diff
@@ -1,11 +1,165 @@
+/* eslint-disable camelcase */
+
 'use strict';
 
 {
+  // #region 1.1_&_1.2
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+    'The 7 Habits of Highly Effective People',
```

Comment:
I know that the instructions and code sample were a bit confusing. The idea was to directly create a list of bookIds here. Just like the example: `harry_potter_chamber_secrets`.